atest/test/02_Content_Keywords/files.robot

```diff
@@ -8,17 +8,35 @@ Test Teardown       Run Keywords    Restore Library Timeout    AND    Wait For A
 ${CUSTOM_DL_PATH} =     ${CURDIR}/download_file
 
 *** Test Cases ***
-Upload File
+Upload upload_test_file
+    Upload Named File   test_upload_file
+
+# 75 starts to fail with firefox and chromium
+# Upload Sized File  75
+# Upload Sized File  512
+# Upload 75MB file
+#    Upload Sized File  75
+
+Upload 1MB file
+    [Tags]    no-windows-support
+    Upload Sized File  1
+    
+Upload 74MB file
+    [Tags]    no-windows-support
+    Upload Sized File  74
```

Comment:
Should there be test which fails on too big file?

Reply:
Implemented at https://github.com/MarketSquare/robotframework-browser/pull/1511 directly against master